README.md

```diff
@@ -24,15 +24,15 @@ You can name the device whatever you want with the `-d` option. If not present,
 ### Associate device to your account
 This will create a new device and associate it to the account for the API key.
 ```
-npx @nrfcloud/device-simulator-v2 -k <api key> -a -t atv2
+npx @nrfcloud/device-simulator-v2 -k <api key> -a -t atv2 -a preconnect
 ```
 
 ### Simulate sensor outputs
 Include any combination of the options listed after `-s` below:
 ```
-npx @nrfcloud/device-simulator-v2 -k <api key> -a -d <device ID you already associated> -s gps,acc,device,temp
+npx @nrfcloud/device-simulator-v2 -k <api key> -a preconnect -d <device ID you already onboarded> -s gps,acc,device,temp
 ```
-*Note! Usually the `-a` is not necessary, since the device is already associated to your account. However, due to bug IRIS-3450, this does not always work. Adding the `-a` is harmless and is a workaround for now.*
+*Note! Usually the `-a` is not necessary, since the device is already onboarded to your account. However, due to bug IRIS-3450, this does not always work. Adding the `-a` is harmless and is a workaround for now.*
```

Comment:
is this still true? doesn't this change the command to now become necessary  (telling the simulator to use the "jitp" or "preconnect" modality). It also seems IRIS-3450 is not reproducible anymore?